packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -318,24 +279,20 @@ function updateContainers(
     }
   });
 
-  // update the value
-  containerGroups = computedContainerGroups;
-}
+  return computedContainerGroups;
+});
+
+onMount(async () => {
+  podUnsubscribe = podsInfos.subscribe(podInfos => {
+    pods = podInfos;
+  });
```

Comment:
Questioning the need for this code, can't we simply use `$podsInfos` instead of pods ? or makes something like the following

```
let pods = $derived($podsInfos)
````

Reply:
Thanks, it simplifies

4a1db8a330f67f4a23b05297a4cc1548990fdac4

I clicked the "Create pod" button and it still seems to work